cypress/integration/patch-route.js

```diff
@@ -161,10 +161,78 @@ describe("Patch route", function() {
           clickSorterAndAssertTasksAreFetched(sortBy)
         );
       });
+
+      it("Fetches additional tasks as the user scrolls", () => {
+        cy.get(".ant-table-row")
+          .invoke("toArray")
+          .then($initialTasks => {
+            cy.get(".ant-table-body").scrollTo("bottom", { duration: 300 });
+            cy.wait(200);
+
+            waitForGQL("@gqlQuery", "PatchTasks");
+            cy.get("@gqlQuery")
+              .its("requestBody.variables.page")
+              .should("equal", 1);
+
+            // confirm additional tasks were appended to table items
+            cy.get(".ant-table-row").should(
+              "have.length.greaterThan",
+              $initialTasks.length
+            );
+          });
+      });
+
+      it("Task count increments by the number of additional tasks fetched", () => {
```

Comment:
the test description implies that the task count increases by an exact number but we are only verifying that the task count increases after we scroll. 
A way to test what the description claims could be to inspect the limit query variable or response length and assert that the new task count is equal to the original task count + response length or limit variable